requirements-base.txt

```diff
@@ -73,7 +74,8 @@ kombu==4.6.11
 # Note, grpcio>1.30.0 requires setting GRPC_POLL_STRATEGY=epoll1
 # See https://github.com/grpc/grpc/issues/23796 and
 # https://github.com/grpc/grpc/blob/v1.35.x/doc/core/grpc-polling-engines.md#polling-engine-implementations-in-grpc
-grpcio==1.35.0
+grpcio==1.35.0; python_version == '3.6'
+grpcio==1.39.0; python_version > '3.6'
```

Comment:
Is there any reason we can't just try and upgrade some of these? These don't _seem_ like they would be terrible right?

Reply:
I didn't want to deal with testing these packages at this moment in order to move quickly.
IMHO, it's OK to diverge temporarily this dev env vs M1's dev env.
I can add it to my post-completion bucket if you wish to.

Reply:
@armenzg I can deal with upgrading grpcio.

Reply:
thank you @joshuarli !